front-end/src/pages/Activities.js

```diff
@@ -0,0 +1,59 @@
+
+
+import React, { Component } from 'react';
+
+//Activities List
+class ActivitiesList extends Component {
```

Comment:
Is it Activities or ActivitiesList?  file name should match class should match export